l10n_mx_toponyms/examples/l10n_mx_estados.xml

```diff
@@ -0,0 +1,575 @@
+        </record>
+        
+        <record id="res_country_state_mx_qr" model="res.country.state">
+            <field name="country_id" ref="base.mx"/>
+            <field name="name">Quintana Roo</field>
+            <field name="code">ROO</field>
+        </record>
+        
+        <record id="res_country_state_mx_slp" model="res.country.state">
+            <field name="country_id" ref="base.mx"/>
+            <field name="name">San Luis Potosí</field>
+            <field name="code">SLP</field>
+        </record>
+        
+        <record id="res_country_state_mx_sin" model="res.country.state">
+            <field name="country_id" ref="base.mx"/>
+            <field name="name">Sinaloa</field>
+            <field name="code">SIN</field>
+        </record>
+        
+        <record id="res_country_state_mx_son" model="res.country.state">
+            <field name="country_id" ref="base.mx"/>
+            <field name="name">Sonora</field>
+            <field name="code">SON</field>
+        </record>
+        
+        <record id="res_country_state_mx_tab" model="res.country.state">
+            <field name="country_id" ref="base.mx"/>
+            <field name="name">Tabasco</field>
+            <field name="code">TAB</field>
+        </record>
+        
+        <record id="res_country_state_mx_tamps" model="res.country.state">
+            <field name="country_id" ref="base.mx"/>
+            <field name="name">Tamaulipas</field>
+            <field name="code">TAM</field>
+        </record>
+        
+        <record id="res_country_state_mx_talx" model="res.country.state">
+            <field name="country_id" ref="base.mx"/>
+            <field name="name">Tlaxcala</field>
+            <field name="code">TLA</field>
+        </record>
+        
+        <record id="res_country_state_mx_ver" model="res.country.state">
+            <field name="country_id" ref="base.mx"/>
+            <field name="name">Veracruz</field>
+            <field name="code">VER</field>
+        </record>
+        
+        <record id="res_country_state_mx_yuc" model="res.country.state">
+            <field name="country_id" ref="base.mx"/>
+            <field name="name">Yucatán</field>
+            <field name="code">YUC</field>
+        </record>
+        
+        <record id="res_country_state_mx_zac" model="res.country.state">
+            <field name="country_id" ref="base.mx"/>
+            <field name="name">Zacatecas</field>
+            <field name="code">ZAC</field>
+        </record>
+        
+        <record model="res.country.state" id="AL">
+            <field name="code">AL</field>
+            <field name="country_id" ref="base.us"/>
+            <field name="name">Alabama</field>
+       </record>
+
+       <record model="res.country.state" id="AK">
+            <field name="code">AK</field>
+            <field name="country_id" ref="base.us"/>
+            <field name="name">Alaska</field>
+       </record>
```

Comment:
@agb80 The US states are added by the base module. If the state code is wrong, it should be fixed there.

If you want to update them, please use `id="base.XX"`